addon/array/filter-by.js

```diff
@@ -7,11 +7,15 @@ const defaultValue = [];
 export default createClassComputed(
   [false, true, false],
   (array, key, value) => {
-    return computed(normalizeArrayKey(array, [key]), value, (array, value) => {
+    let args = [normalizeArrayKey(array, [key])];
+    if (value) {
+      args.push(value);
+    }
```

Comment:
In this case, `value` is not yet resolved, so it is either a string key, or undefined if not supplied. For example, if you wrote `filterBy('array', 'key', false)`, `value` would be the string "nonStrings.2".